ios/CodePush/CodePush.m

```diff
@@ -480,6 +480,13 @@ - (void)savePendingUpdate:(NSString *)packageHash
         return;
     }
     
+    if (isRunningBinaryVersion) {
+        // This only matters in Debug builds. Since we do not clear "outdated" updates,
+        // we need to indicate to the JS side that somehow we have a current update on
+        // disk that is not actually running.
+        [package setObject:@(isRunningBinaryVersion) forKey:@"_isDebugOnly"];
```

Comment:
Would it be easier to read this if we just set it to true instead of relying on the `isRunningBinaryVersion` variable?